build.wake

```diff
@@ -0,0 +1,73 @@
+tuple DevicetreeOverlayGeneratorOptions =
```

Comment:
In our typical naming convention, we tend to name these things with a suffix of `Plan`, not `Options`, for example [`BitstreamPlan`](https://github.com/sifive/api-generator-sifive/blob/0b554f9bc40c31ca03cdeb7c5e2f3b3cd913b22c/fpga/mcs.wake#L1).

That said, I think that this is a perfectly clear name.